src/renderer/components/auth/ProfileBox.tsx

```diff
@@ -154,7 +154,7 @@ const ProfileBox = ({
           </div>
         </div>
 
-        <p className="text-sm">{user.name}</p>
```

Comment:
instead of slice, `text-ellipsis overflow-hidden` is probably more convenient :)